ipv-core/src/main/java/com/ibm/whc/deid/util/CountryManager.java

```diff
@@ -29,17 +29,17 @@
     implements Serializable {
   /** */
   private static final long serialVersionUID = -1974159797966269524L;
-  protected final Resources resourceType = Resource.COUNTRY;
-  protected final Collection<ResourceEntry> resourceCountryList =
+  protected transient final Resources resourceType = Resource.COUNTRY;

```

Comment:
All the implementations of Resources are enums, so they are Serializable - probably just a matter of declaring Resources interface to be serializable.